.github/workflows/linux_conditional_compilation.yml

```diff
@@ -0,0 +1,134 @@
+name: Linux Conditional Compilation (Ubuntu 22.04, Python 3.11)
+on:
+  workflow_dispatch:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+#  pull_request:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#  push:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#    branches:
+#      - master
```

Comment:
remove commented code?

Reply:
Currently, all _from-Azure_ pipelines except for the `linux.yml` are expected to execute only once a day so that these triggers are commented out. Once we use not-GitHub runners, they will be uncommented.